src/widget/form/chatform.cpp

```diff
@@ -162,7 +162,8 @@ void ChatForm::onTextEditChanged()
 
 void ChatForm::onAttachClicked()
 {
-    QStringList paths = QFileDialog::getOpenFileNames(0,tr("Send a file"));
+    QStringList paths = QFileDialog::getOpenFileNames(this,
+                                                      tr("Send a file"));
```

Comment:
The newline here is maybe a bit gratuitous, since it only saves 5 characters, and there are many longer lines around it...